avm/res/insights/webtest/main.bicep

```diff
@@ -0,0 +1,201 @@
+metadata name = 'Web Tests'
+metadata description = 'This module deploys a Web Test.'
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Required. Name of the webtest.')
+param name string
+
+@sys.description('Required. Resource ID of the App Insights resource to link with this webtest.')
+param appInsightResourceId string
+
+@sys.description('Required. User defined name if this WebTest.')
+param webTestName string
+
+@sys.description('Optional. Resource tags. Note: a mandatory tag \'hidden-link\' will be automatically added to the tags defined here.')
```

Comment:
Sounds good. The only thing is add is a reference to the parameter the value is taken from, i.e., the resource Id